packages/core/src/utils/variant-group.ts

```diff
@@ -17,18 +17,29 @@ interface VariantGroup {
   items: HighlightAnnotation[]
 }
 
-export function parseVariantGroup(str: string | MagicString, separators = ['-', ':'], depth = 5) {
+export function parseVariantGroup(str: string | MagicString, separators = ['-', ':'], depth = 5, onlyAttribute?: boolean) {
   const regexClassGroup = makeRegexClassGroup(separators)
   let hasChanged
   let content = str.toString()
   const prefixes = new Set<string>()
   const groupsByOffset = new Map<number, VariantGroup>()
-
+  const processedStr = (str: string) => str.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')
+  const isAttributor = (matchText: string, offset: number) => {
+    const reg = new RegExp(`<[\\w_\\-]+[^>\\/]*${processedStr(matchText)}[^>\\/]*\/?>`, 'gm')
```

Comment:
I am not sure about dynamic constructed RegExp, it would hurt the performance and also bring potential security concerns when we construct it from user input

Reply:
I tried to write a static regular expression for processing, and it seemed to pass all the tests. What do you think?